### ISSUE TITLE
Joycon input is not being received

### ISSUE THREAD

**Issue by Swarkin** [defold/defold#6637]
**Describe the bug (REQUIRED)**
Joycon input is not being received at all in Defold.

**To Reproduce (REQUIRED)**
1. Play any defold game and try to use Joycons.

**Expected behavior (REQUIRED)**
Input being received normally like any other controller

**Defold version (REQUIRED):**
  - Version 1.3.2

**Platforms (REQUIRED):**
 - Platforms: Windows
 - OS: Windows 10

**Additional context (OPTIONAL):**
Using the website https://gamepad-tester.com/ I can see that every button does work, so it has to be a defold issue

**Comment by subsoap** [comment]
Are you on Windows? Please try adding a Windows version of the input for the default gamepad input file and see if it works then.

**Comment by subsoap** [comment]
@britzl Was this fixed? Last time I tried joycons it wasn't work either.

**Comment by britzl** [comment]
No, I don't believe it has been fixed. It seems like @Swarkin closed this issue. Perhaps it was enough to add a windows version of the gamepad bindings for JoyCons?

**Comment by subsoap** [comment]
> No, I don't believe it has been fixed. It seems like @Swarkin closed this issue. Perhaps it was enough to add a windows version of the gamepad bindings for JoyCons?

When I tried to get it to work I did that with no luck.

**Comment by subsoap** [comment]
@Swarkin

Why did you close this issue?

**Comment by Swarkin** [comment]
I thought the issue was dead or forgotten because there was no activity at all for multiple months, I don't use Defold anymore so I couldn't test anything either 😅
I'll reopen the issue but I can't really help with fixing it

**Comment by subsoap** [comment]
> I thought the issue was dead or forgotten because there was no activity at all for multiple months, I don't use Defold anymore so I couldn't test anything either 😅 I'll reopen the issue but I can't really help with fixing it

It may be low priority relative to other things but doesn't mean it won't one day be solved! It may require a special driver to work on Windows properly though and that might require a native extension.

**Comment by Jhonnyg** [comment]
This should be fixed by now.
